test/tests/variables/previous_newton_iteration/test.i

```diff
@@ -10,6 +10,10 @@
   elem_type = QUAD9
 []
 
+[Problem]
+  previous_nl_solution_required = true
```

Comment:
Because now I use tag to manage this solution vector, we have to add the vector earlier. This requires users to turn this flag to true when the vector is used. But I think this is the right design. Users can use their own actions to add the vector for better user interface if desired.

Reply:
Looks like no application is using this. Surprising.